doc/src/tutorial/001_adding_lalrpop.md

```diff
@@ -56,8 +56,9 @@ hand and use it with LALRPOP, then check out the [lexer tutorial].)
 [the next section]: 002_paren_numbers.md
 [lexer tutorial]: ../lexer_tutorial/index.md
 
-Next we have to add `build.rs` itself. This should just look like the
-following:
+Next we have to add `build.rs` itself. For those unfamiliar with this feature, the `build.rs` file
+should be placed next to your `Cargo.toml` file and not inside the `src` folder with the rest of
+your Rust code. This should just look like the following:
```

Comment:
I'm wondering if it would be possible to link to an appropriate place in the Cargo guide here 🤔

Reply:
It turns out there is a link a few lines above but I'm not against making the first `build.rs` in this paragraph a link to the cargo page too (https://doc.rust-lang.org/cargo/reference/build-scripts.html)

Reply:
Thanks for the thought. I've added a commit to add more links to the cargo docs for the build.rs. I don't know when enough is enough but I am certainly happy to err on having too many than too few.